source/integration_tools.html.md

```diff
@@ -0,0 +1,73 @@
+# Integration Tools
+
+Integration Tools is a section of Workshop (Welkin's codeless configuration tool), which allows you to configure Apps and other integrations. You can configure which systems have access to data within Welkin and how they access it.
+
+# Overview
+
+Integration Tools allows developers and program directors to manage their integrations with Welkin without contacting a member of the Welkin professional services team. This is a self serve tool for managing how data flows out of Welkin via Welkin's APIs.
+
+Integration Tools is broken into 3 main sections. Credentials for setting up access credentials. Webhooks for getting notifcations about changes to data in Welkin. Environments for selecting which Welkin instance/environment you want to configure integrations for.
+
+# Credentials
+
+Credentials issued via Integration Tools grants Apps and Integrations access to all the data in the Welkin environment. These `client_id` and `client_secret` pair generated by Integration Tools is for syncing data for your entire Welkin environment and is not scoped to specific patients or workers within your Environment.
+
+Each App or Integration you configure for your Welkin Environment should have its own set of credentials for accessing data. Granting separate credentials for each system which has access to Welkin data ensures that if you want to disable a single system you can do this without effecting others.
+
+Credentials created in Integration Tools are active as soon as they are created and can be used to generate access tokens. Access tokens are short lived and are used to make calls to the API. Details on this token exchange process is [documented here](/data_api.html/#authentication).
+
+<aside>When you invalidate Credentials in Integration Tools, Access Tokens are NOT invalidated. They will expire as normal. The credentials will no longer be valid for generating new Access Tokens.</aside>
+
+# Webhooks
+
+Webhooks specified via Integration Tools send notifications to Apps and Integrations when data is updated in Welkin.
+
+In Integration Tools you select the destination for these webhooks, the method used to secure the data transfer of webhooks, and the credentials used in the selected security method.
+
+When Welkin adds new API endpoints we will not automatically opt in notifications to be sent for those new data types. You must explicitly opt in for each type of notification you want to receive.
+
+You can pause webhooks from this page which prevents us from sending webhooks to your system.
+
+# Environments vs Configurations
```

Comment:
This section is fairly abstract -- it might be helpful to connect this to the "test" and "live" environments that most customers have